cachito/workers/pkg_managers/gomod.py

```diff
@@ -228,12 +312,47 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
             "GOPROXY": f"{athens_url}|{athens_url}",
             "PATH": os.environ.get("PATH", ""),
             "GOMODCACHE": "{}/pkg/mod".format(temp_dir),
+            "GOTOOLCHAIN": "local",
         }
         if "cgo-disable" in request.get("flags", []):
             env["CGO_ENABLED"] = "0"
 
         run_params = {"env": env, "cwd": app_source_path}
 
+        go = Go()
+        go1_21 = pkgver.Version("1.21")
+        go_base_version = go.version
+        go_mod_version_msg = "go.mod recommends/requires Go version: {}"
+
+        if (_ := _get_gomod_version(git_dir_path)) is None:
```

Comment:
Very nitpickish comment, but I'd rather that this `_` variable had a name, since it's being used outside of the if clause context.